LearningAppStoryboardSegueOptionsUIKit/LearningAppStoryboardSegueOptionsUIKit/StoryboardSegueOption.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  StoryboardSegueOption.swift
+//  LearningAppStoryboardSegueOptionsUIKit
+//
+//  Created by Saurabh Verma on 13/12/24.
+//
+
+import Foundation
+
+enum StoryboardSegueOption: String, CaseIterable, Identifiable {
+    
```

Comment:
This needs unit tests. Also add TODO for remaining cases examples which will be taken up in future PRs.

Reply:
Added TODOs